### ISSUE TITLE
Increasing mp_coop_lobbysize doesn't increase amount of squads.

### ISSUE THREAD

**Issue by MaxMatti** [jaredballou/insurgency-sourcemod#37]
I installed sourcemod build 6015 (the newest version that doesn't crash on linux) and copied the "insurgency" and "cooplobby" mod files from the repo to the corresponding locations in my server folder.
Issuing the command `mp_coop_lobbysize 32` increases the player limit on my coop server to 32 but there is still only one squad, so only 8 players can actually play. Did I miss something or do something wrong?

**Comment by jaredballou** [comment]
No, you also need to have a custom theater which supports thew additional players.
